README.md

```diff
@@ -112,70 +117,112 @@ A `.env` file should be created in the repository root, to keep configuration de
 cp example.env .env
 ```
 
-Replace `REPO_PATH_SERVER` with the path from above.
+```
+cd workspace
+pwd
+cd ..
+```
+Please note the printing of the pwd command as `STABLE_ROOT_PATH`.
+
+Please adjust in the `.env` file at least the `REPO_PATH_SERVER` with the value from #1 and the `STABLE_ROOT_PATH` from the last command above.
+If you have apps you like to mount to all running nextcloud versions in this setup, just set the `ADDITIONAL_APPS_PATH`. 
+Note, this is not recommended for apps that have individual versions for different server versions. But that could be a good place to mount your apps you are working on to test them accross different server versions.
+
 
 ##### 4. Running different stable versions
 
 The docker-compose file provides individual containers for stable Nextcloud releases. In order to run those you will need a checkout of the stable version server branch to your workspace directory. Using [git worktree](https://blog.juliushaertl.de/index.php/2018/01/24/how-to-checkout-multiple-git-branches-at-the-same-time/) makes it easy to have different branches checked out in parallel in separate directories.
 
+**Server part**
+
+*Example for Nextcloud Server version 25*
+
 ```
 cd workspace/server
-git worktree add ../stable23 stable23
+git worktree add ../stable25 stable25
+cd ../..
 ```
-As in the in the `server` folder, the `3rdparty` submodule is needed:
+
+As in the `server` folder, the `3rdparty` submodule is needed:
 ```
-cd ../stable23
+cd workspace/stable25
 git submodule update --init
+cd ../..
 ```
 
-The same can be done for stable24, stable25... and so on. 
+The same can be done for stable24, stable25... and so on.
+Minimum version is stable16.
+
+
+**Apps part**
 
 Git worktrees can also be used to have a checkout of an apps stable brach within the server stable directory.
 
 ```
 cd workspace/server/apps-extra/text
-git worktree add ../../../stable23/apps-extra/text stable23
+git worktree add ../../../stable25/apps-extra/text stable25
+cd ../../../..
 ```
 
 Since the viewer app is kind of required for Nextcloud server, you should also add that to the stable worktrees:
 
 ```
 cd workspace/server/apps/viewer
 git worktree add ../../../stable25/apps/viewer stable25
+cd ../../../..
 ```
 
+It is recommanded to use the `apps-extra` folder for apps that have individual branches for different server versions. For apps that are mutiple server versions supporting you you can use the `apps-shared` folder. It will be mountet to all stable versions. Just set it in the `.env` file from #3.
```

Comment:
```suggestion
It is recommanded to use the `apps-extra` folder for apps that have individual branches for different server versions. For apps that are mutiple server versions supporting you you can use the `apps-shared` folder. It will be mountet to all stable versions. This folder is located in `data/apps-shared` by default.
```